paper/paper.md

```diff
@@ -25,47 +25,63 @@ bibliography: paper.bib
 
-`pyopmspe11` is a simplified and flexible Python tool for the three cases in the SPE Comparative Solution
-Project via configuration files. `pyopmspe11` relies on the OPM Flow numerical simulator [@Rassmussen:2021], where the
+
+
+Based on the acquired knowledge by developing the aforementioned tools, as well as from using/contributing to other open-source projects, then we have developed and made open the `pyopmspe11` tool which facilitates reproducible solutions to the SPE11 benchmark, which focus on GCS at different scales [@Nordbotten:2024]. 
+A previous benchmark study for GCS can be found in @Class:2009. One key differece of the SPE11 benchmark from the previous one is that no grids were given in the description, i.e., one of the main task for the participants was to create suitable grids (e.g., structured grids such as Cartesian or unscrtuctured grids such as corner-point grids). The participants were encouraged to share data (e.g., input decks, code, submitted results), with the opportunity to store the data for open access. This is where developing tools that made all steps reprodubible (i.e., preprocessing and postprocessing) become handy, and for this benchmark study, one available tool is `pyopmspe11`.
+
+
+
+`pyopmspe11` is a simplified and flexible Python tool to execute the three cases in the SPE Comparative Solution
+Project using configuration files. `pyopmspe11` relies on the OPM Flow numerical simulator [@Rassmussen:2021], where the
 implementation of the CO$_2$ model can be found in @Sandve:2021. The primary contribution of `pyopmspe11` lies in its 
 data preprocessing and postprocessing capabilities. It offers flexibility in generating various types of grids, 
-including Cartesian, tensor, and corner-point grids. These grids adhere to standard industry formats, making them 
-compatible not only with OPM Flow but also with other simulators. Additionally, `pyopmspe11` supports different resolutions, 
-having been tested to generate approximately 160 million cells. In the context of data postprocessing, `pyopmspe11` not only 
+including Cartesian, tensor, and corner-point grids. These grids adhere to the standard industry format (i.e., Eclipse grid format), making them 
+compatible not only with OPM Flow but also with other simulators. Here, we mention two existing widely-use visualization/postrocessing software for OPM Flow: [ParaView](https://www.paraview.org) and [ResInsight](https://resinsight.org). While these tools are very useful, to the authors knolwedge, there is no existing functionality in these tools to handle all the necessary postporcessing to generate all data reporting as required in the SPE11 benchmark study. For example, to write the csv maps of the quantities from a simulation grid to a given reporting grid, where both grids do not overlap, or to compute the convective mixing as defined in Eq. (17) in [@Nordbotten:2024]. 
```

Comment:
```suggestion
compatible not only with OPM Flow but also with other simulators. Here, we mention two existing widely-use visualization/postprocessing software for OPM Flow: [ParaView](https://www.paraview.org) and [ResInsight](https://resinsight.org). While these tools are very useful, to the authors knowledge, there is no existing functionality in these tools to generate all postprocessed data as required in the SPE11 benchmark study. For example, to write the csv maps of the quantities from a simulation grid to a given reporting grid, where both grids do not overlap, or to compute the convective mixing as defined in Eq. (17) in [@Nordbotten:2024]. 
```

Not sure the last sentence is quite correct (you can do a lot of interpolation to other specified grids and/or complex calculations (even embedded python functions) inside Paraview. However, I of course see the benefit of having all that already readily created in a specialized tool, so feel free to keep the sentence.